packages/decoder/lib/Web3Adapter.ts

```diff
@@ -0,0 +1,50 @@
+import Web3Eth from "web3-eth";
+type PastLogsOptions = {
+  toBlock?: string | number;
+  fromBlock?: string | number;
+  address?: string | string[];
+}
```

Comment:
This should probably not live amongst the imports; if it's not getting exported, maybe move it to the end of the file?